tests/test_misc.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from chaostoolkit.cli import is_yaml
```

Comment:
Ordering usually is as follows:

```
stdlib

dependencies

local
```

so here:

```
import pytest

from chaostoolkit.cli import is_yaml
```

Reply:
done